ddtrace/tracer/option.go

```diff
@@ -105,6 +105,28 @@ func defaults(c *config) {
 	if v := os.Getenv("DD_ENV"); v != "" {
 		WithEnv(v)(c)
 	}
+
+	if v := os.Getenv("DD_SERVICE_NAME"); v != "" {
```

Comment:
Sorry for the back and forth, but we were just in the process of standardising these values. The env. var. we will be using is `DD_SERVICE`. Can you please change it?